.github/workflows/k8s-content-pr.yaml

```diff
@@ -101,16 +101,24 @@ jobs:
       - container-main
       - get-pr-number
     runs-on: ubuntu-latest
-    name: Comment on the PR
+    name: Upsert comment on the PR
     steps:
-      - uses: actions/github-script@v7
+      - uses: thollander/actions-comment-pull-request@v2
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            await github.rest.issues.createComment({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              issue_number: ${{ needs.get-pr-number.outputs.pr-number }},
-              body: ':robot: The image for this PR is available at:
-            `ghcr.io/complianceascode/k8scontent:${{ needs.get-pr-number.outputs.pr-number }}`'
-            });
+          message: |
+            :robot: A k8s content image for this PR is available at:
+            `ghcr.io/complianceascode/k8scontent:${{ needs.get-pr-number.outputs.pr-number }}`
+
+            <details>
+            <summary>Click here to see how to deploy it</summary>
+
+            If you alread have Compliance Operator deployed:
+            ```utils/build_ds_container.py -i ghcr.io/complianceascode/k8scontent:${{ needs.get-pr-number.outputs.pr-number }}```
```

Comment:
I did a double take here because I'm so used to using this script to build fresh content from source locally, and I thought "Wait! Do we need to do this?"

Then I saw the changes below :)

Perhaps our tooling needs to evolve.